commands/filescommands/commands.go

```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "container",
-			Usage:       "Used for Cloud Files container operations",
+			Usage:       "Storage compartments for your objects/files.",
```

Comment:
I was trying my hardest to not be cyclic, and I stole it straight from Rackspace's docs about containers.